### ISSUE TITLE
A Spanish language category within users.rust-lang.org

### ISSUE THREAD

**Issue by mariomartinezsz** [rust-community/team#291]
Hola, amigos. I'm Mario, a member of [Rustlang_mx](https://twitter.com/rustlang_mx).

In addition to the Twitter and Meetup accounts, we have been using a [Telegram](https://t.me/RustMX) group to organize events and for Spanish speakers to find a sense of community.

We are already receiving technical questions regularly in the Telegram group. And some participants are proposing to create a Forum.

I have suggested to my colleagues we can talk first with Rust Community in order to find an official space for the Spanish speaking community.

Do you think it would be possible to create a forum category for Spanish speakers?

**Comment by Manishearth** [comment]
We'd need to be able to moderate these according to the code of conduct, and pay attention to them. I think for now it would be good for y'all to set up an unofficial forum for this, new official forums are a lot of work.

**Comment by sebasmagri** [comment]
@Manishearth yes, the moderation bit was the first thing I mentioned. However, this might be a good opportunity to get more local contributors involved in the project by expanding the moderators team for language specific members.

Creating and maintaining a new forum is also quite some work and would also require moderation. The differences in the moderation policies might introduce confusion, and could even set a precedent for other communities which would prefer to be under the umbrella of the Rust Project of not being welcomed. Also in some cases this might have prohibiting cost requirements for folks around the globe.

**Comment by Manishearth** [comment]
I don't think it's that easy. Firstly, the general-user forums like users.rust-lang.org and #users on Discord are somewhat exceptions, by and large we try to build platforms for team work, not for general discussion. General discussion is _way_ harder to moderate, for example almost all of the moderation stuff the Discord mods deal with are in #users. I would rather suggest folks start building a community on Spanish Stack Overflow, or use some free forum software. Stack Overflow has problems, but in my experience if a particular niche (e.g. the Rust niche on the English stack exchange) is full of folks who care, most of those problems go away.

Secondly, moderation is kinda hard to grow, I want us to be better at that, but growing it cross-language is certainly a challenge, and it's one we've been super careful of. I'd rather see such forums start as unofficial venues, and after a while if they still see value in being official, then we can work it out, and perhaps pull from the existing set of mods for the venue.

**Comment by jrvidal** [comment]
FWIW there is already a _non-official_ Rust Lang en Español Discord server https://discord.gg/swXECAkU

**Comment by skade** [comment]
The linked issue doesn't have a lot, but there have been previous discussions on such a feature in 2017. https://github.com/rust-community/team/issues/125

A well-tended discourse with such features can be seen here: https://discourse.mozilla.org/

**Comment by sebasmagri** [comment]
> FWIW there is already a _non-official_ Rust Lang en Español Discord server https://discord.gg/swXECAkU

Yes, multiple similar channels exist on Telegram, as described by Mario, and over IRC/Matrix as well. The issue in discussion is that after fostering participation in those venues the mentions of the necesity for a more _forum_ alike discussion medium has been increasing.

**Comment by sebasmagri** [comment]
> The linked issue doesn't have a lot, but there have been previous discussions on such a feature in 2017. #125
> 
> A well-tended discourse with such features can be seen here: https://discourse.mozilla.org/

Thanks so much Florian, had been looking this one in the wrong places.
